crypto/src/proofs/transparent.rs

```diff
@@ -737,16 +735,23 @@ impl SwapProof {
             return Err(anyhow!("transmission key mismatch"));
         }
 
-        // Value commitment integrity.
-        if value_1_commitment != -self.value_t1.commit(self.delta_1_blinding) {
-            return Err(anyhow!("value commitment mismatch"));
-        }
+        // TODO: no value commitment checks until flow encryption is available
+        // // Value commitment integrity.
+        // if value_1_commitment != -self.value_t1.commit(self.delta_1_blinding) {
+        //     return Err(anyhow!("value commitment mismatch"));
+        // }
 
-        if value_2_commitment != -self.value_t2.commit(self.delta_2_blinding) {
-            return Err(anyhow!("value commitment mismatch"));
-        }
+        // if value_2_commitment != -self.value_t2.commit(self.delta_2_blinding) {
+        //     return Err(anyhow!("value commitment mismatch"));
+        // }
 
-        if value_fee_commitment != -self.value_fee.commit(self.fee_blinding) {
+        let pen_denom = asset::REGISTRY.parse_denom("upenumbra").unwrap();
```

Comment:
This should use `STAKING_TOKEN_ASSET_ID` instead, to avoid hardcoding the denom.